style.css

```diff
@@ -65,4 +65,328 @@
+    margin-bottom: 15px;
+}
+
+.descriptions {
+    font-size: 14px;
+    line-height: 24px;
+    font-weight: 400;
+}
+
+.right-side {
+    position: relative;
+}
+
+.right-side__main-photo {
+    margin: 100px 50px;
+}
+
+.right-side__square {
+    position: absolute;
+    bottom: 200px;
+    right: 0;
+}
+
+.our-team {
+    height: 100vh;
+    width: 100%;
+    color: white;
+    background: url("./images/team-bg.png");
+    background-size: cover;
+}
+
+.our-team__title {
+    text-align: center;
+    font-weight: 700;
+    font-size: 12px;
+    line-height: 100px;
+    letter-spacing: 6px;
+}
+
+.our-team__left-side {
+    position: relative;
+}
+
+.our-team__left-side__square {
+    position: absolute;
+    top: -90px;
+    left: -90px;
+}
+
+.our-team__right-side {
+    margin: 150px 100px 217px 72px;
+    text-align: center;
+}
+
+.sides {
+    display: flex;
+}
+
+.booking {
+    height: 100vh;
+    width: 100%;
+}
+
+.booking__left-side {
+    margin: 140px 40px 0 70px;
+    text-align: center;
+}
+
+.booking__left-side__line {
+    margin-bottom: 20px;
+}
+
+.form__inputs input {
+    background: #E5E5E5;
+    border-radius: 5px;
+    height: 60px;
+    width: 276.8px;
+    border: none;
+    margin-right: 10px;
+    margin-bottom: 10px;
+    padding-left: 15px;
+}
+
+.btn__book-now {
+    position: absolute;
+    left: 95px;
+    text-transform: uppercase;
+    font-size: 12px;
+    line-height: 20px;
+    height: 60px;
+    width: 605px;
+    border: none;
+    color: white;
+    border-radius: 5px;
+    background: #E8C300;
+}
+
+.booking__right-side {
+    position: relative;
+}
+
+.booking-right-side__square {
+    position: absolute;
+    top: -90px;
+    right: 250px;
+}
+
+.booking__title {
+    margin-top: 30px;
+    text-align: center;
+    font-weight: 400;
+    font-size: 12px;
+    line-height: 12px;
+    color: rgba(51, 51, 51, 0.5);
+}
+
+.booking__title span {
+    font-weight: 700;
+}
+
+.specialties {
+    height: 100vh;
+    width: 100%;
+    color: white;
+    background: url("./images/specialty's-bg.png");
+    background-size: cover;
+}
+
+.specialties__title {
+    text-align: center;
+    font-weight: 700;
+    font-size: 12px;
+    line-height: 100px;
+    letter-spacing: 6px;
+    text-transform: uppercase;
+}
+
+.specialties__left-side {
+    position: relative;
+}
+
+.specialties__left-side__square {
+    position: absolute;
+    top: -90px;
+    left: -90px;
+}
+
+.specialties__right-side {
+    margin: 150px 100px 217px 72px;
+    text-align: center;
+}
+
+.gallery {
+    display: flex;
+    justify-content: space-between;
+    height: 100%;
+}
+
+.gallery-photo {
+    height: 337.5px;
+    width: calc(100%/4);
+}
+
+.contact {
+    text-align: center;
+    margin-top: 63px;
+    height: 100vh;
+    width: 100%;
+}
+
+.contact-subtitle {
+    margin-bottom: 50px;
+}
+
+.contact-form {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+
```

Comment:
the same with contact form with button
all inputs and button are centred, but visually it looks strange, because a button is smaller than textarea, and textarea is bigger than previous inputs

make all these fields with button the same size